### PR TITLE
gestion des resulats de recherche en utilisant les parametres des cha…

### PR DESCRIPTION
**Pull request by cholletNovius**
result Request sql with the "allowable_chars" config.

### CODE REVIEW

**Comment by SwithFr** [comment]
:+1:

**Comment by smeeckaert** [comment]
It looks like `allowable_chars` is set to `''` by default, this modification would potentially change the behaviour of the `split_string` method for instances called without the `allowable_chars` parameter.

**Comment by cholletNovius** [comment]
modif update

**Comment by cholletNovius** [comment]
up

File: classes/search.php [hunk 1]
```diff
@@ -293,11 +293,16 @@ public static function generate_keywords($keywords, $params)
             'min_word_len' => 2,
             'max_keywords' => 5,
         );
+
+        //If allowable_chars is not in Config, with do a default allowable_chars
+        if(!isset($params['allowable_chars'])) {
+            $params['allowable_chars'] = '*?';
```
Comment (Meroje) [inline classes/search.php]: There's `$default_params` above just for that purpose

**Comment by cholletNovius** [comment]
up

**Comment by Meroje** [comment]
:+1: lgtm

**Comment by Seb-C** [comment]
:+1:

**Pull request merged**